package.json

```diff
@@ -26,6 +26,7 @@
   },
   "license": "MIT",
```

Comment:
If you think that many users will not need to install `gatsby-plugin-image` and you want to dynamic import the module inside of function calls, then a good place to log this warning would be in the gatsby node api `onPreInit` since this is the very first api that runs when bootstrapping.